src/context/app_context.rs

```diff
@@ -19,6 +19,8 @@ pub struct AppContext {
     pub quit: QuitType,
     // event loop querying
     pub events: Events,
+    // choose file instead of opening it
+    pub choosefiles: bool,
```

Comment:
Instead of having a `choosefiles` boolean here, I think we add a new variant to `QuitType` called `ChooseFiles` or something. That was kind of the idea of what `QuitType` is for; tell Joshuto to do things on quit.

Here, we can also add `Args` as a field of `AppContext`. So later on in `open_file.rs` we can check
`if context.args.choosefiles.is_some()`